lib/pharos/host/el7/scripts/install-kubelet.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+. /usr/local/share/pharos/util.sh
+
+set -e
+
+versionlock="/etc/yum/pluginconf.d/versionlock.list"
+linefromfile "^0:kubelet-" $versionlock
+linefromfile "^0:kubectl-" $versionlock
+linefromfile "^0:kubeadm-" $versionlock
+
+yum install -y kubelet-${KUBE_VERSION} kubectl-${KUBE_VERSION} kubeadm-${KUBEADM_VERSION}
```

Comment:
The script name, and the configurer interface actually too, is slightly misleading as we install all kube components here and not just kubelet.

Reply:
Do you have any suggestions for the interface name?

Reply:
`install_kube_packages`?
`install_kube_components`?

Reply:
Fixed.